pandas/core/ops/docstrings.py

```diff
@@ -33,9 +33,13 @@ def _make_flex_doc(op_name, typ):
             desc=op_desc["desc"],
             op_name=op_name,
             equiv=equiv,
-            reverse=op_desc["reverse"],
             series_returns=op_desc["series_returns"],
         )
+        if op_desc["reverse"]:
+            doc_see_also = _see_also_reverse_SERIES.format(
+                reverse=op_desc["reverse"], see_also_desc=op_desc["see_also_desc"],
+            )
+            doc_no_examples = doc_no_examples + doc_see_also
```

Comment:
I think the examples part of the doc building is already to complex to mix the see also in here.

Can you move this condition after line 13, and have something like:
```python
if op_desc["reverse"]:
    base_doc += _see_also_reverse_SERIES.format(...)
```
I think this will be clearer this way.